package.json

```diff
@@ -19,5 +19,12 @@
     "branches": [
       "main"
     ]
+  },
+  "engines": {
+    "node": ">=18"
+  },
```

Comment:
There is no runtime code in this repository, I don't think this is necessary.

Reply:
it would result a warning when installing on node versions lower than 18

Reply:
Since there's no runtime code, only JSON, it's universal.

Reply:
makes no sense, but ok.